### ISSUE TITLE
[CAIA A11y Research]: Debt Resolution, Enhanced Financial Status Report

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#58211]
### Who's on your team?

```bash
Team name: Debt Resolution
OCTO-DE product owner: Denise Coveyduc
Delivery manager: Heather Rienks
Product manager: Tom Davis
Researcher: Charlotte Cesana
Designer/Researcher: Megan Gayle
Designer: Joe Lee
FE engineers: Andrew Rodiek, Brandyn Sullins, Aaron Ponce
BE engineers: Scott James, Kevin Suarez
Product/team Slack channel: #benefits-debt-resolution
...
```

### Do you have an accessibility champion on your team?

No, we don't have a champion (yet)

### Is anyone on your team interested in becoming an accessibility champion?

_Charlotte Cesana_
_Megan Gayle_

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the [CAIA capacity calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/newyork) and can confirm specialists are available to support in our research timeframe
    - **Megan Gayle:** Reviewed the capacity calendar and looks like there is availability at all times except 4pm and the last 15 minutes of the 1pm session
    - :calendar: **Sara Smith:** Once we have the times @mgleo we can see if any of those other sessions will work with schedules. 

## Testing Timeframe

- May 17-May 24, 2023
- 11 a.m., 1 p.m. and 4 p.m. EST
  
### Session Invitations

- Be sure to include the a11y representative(s) listed in this ticket on the session invite, that includes the link. 
- A11y team members will put a a hold on the calendar, but they will need an invitation and link to assist/observe the session. 

### Confirmed sessions
_**NOTE:** Original Participant 1 was not using AT, despite checking box on the screener. The information received from P1 is being moved to the comments, and has been removed from the ticket body, as of May 22, 2023 at 11:12 a.m._ 

#### Participant 1
- **Session Date/Time: xxxx**  
    - @mgleo can you provide us with the following on the participant, when it becomes available? 
        - Assistive Tech:
        - AT Skill Level: 
        - Operating System:
        - Browser:
        - Device Type: Smartphone 
    - [ ] Invite a11y specialist(s) on Zoom invite.

#### Participant 2
- Additional session pending.
- [ ] Invite a11y specialist(s) on Zoom invite.

#### Participant 3
- Additional session pending.
- [ ] Invite a11y specialist(s) on Zoom invite.

## Supporting artifacts (if any)
- [ Prototype Staging URL (5/19/2023)](https://staging.va.gov/manage-va-debt/request-debt-help-form-5655/)
- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_fsr/product_outline.md)
- [User flows](Insert URL here between the parenthesis)
- [Prototype or mockups](https://preview.uxpin.com/f43733391f52c409c4d3152e9a4b101a392e29df#/pages/161460688)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_fsr/research/enhanced_fsr/research-plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_fsr/research/enhanced_fsr/conversation-guide.md)
- [Epics/issue that may be helpful](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/28318)

## Next steps

**Complete these after submitting this ticket with your CAIA specialist**

- [x] Post a link to this ticket in the [accessibility-help Slack channel and tag Terry Nichols](https://dsva.slack.com/archives/C8E985R32).
- [x] Pilot session for AT users (SK/SS) 5/19/2023
- [ ] Your team's researchers and observers [read through platform guidance on research with assistive technology users](https://depo-platform-documentation.scrollhelp.site/research-design/research-with-assistive-technology-users)
- [ ] CAIA Specialists are assigned to your research
- [ ] CAIA Specialists review your conversation guide and research plan
- [ ] Provide specialists your participant details like skill level, operating system, assistive tech used, browser
- [ ] CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future.
- [ ] CAIA Specialists create a[ research notes template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/research/accessibility-findings-template.md)
- [ ] CAIA Post research, specialists will add accessibility findings to your research folder

**Comment by sara-amanda** [comment]
@mgleo While we wait for the sessions to be scheduled, I added **"Next Steps"** to the ticket for you. [Here is a link to the platform guidance on research with assistive technology users](https://depo-platform-documentation.scrollhelp.site/research-design/research-with-assistive-technology-users) to reference.

**Comment by sara-amanda** [comment]
@coforma-terry  I assigned the first session to myself. Should we get a screen reader, perhaps we can also invite @AngelaFowler82 ?

**Comment by AngelaFowler82** [comment]
I am available to support with any screen reader users.

**Comment by sara-amanda** [comment]
## CAIA Pilot Session Notes/Comments
### Meeting 5/19/2023
@charlottecesana @josephrlee @SarahKay8 @sara-amanda 
### Staging Link
https://staging.va.gov/manage-va-debt/request-debt-help-form-5655/
### Assistive Technology Sessions
Review the [documentation on assistive technology sessions](https://depo-platform-documentation.scrollhelp.site/research-design/research-assistive-technology-sessions). 
### Wizard is being used.
- DST URL: https://design.va.gov/patterns/wizards 
- The [Wizard pattern is now deprecated](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/399) and replaced by the the [help users complete a sub-task pattern](https://design.va.gov/patterns/help-users-to/complete-a-sub-task).
### Automatically saves
- **Test auto-save placement**, for logical reading order, to make sure it is not bypassed. 
### VA Profile
- **Updates your overall VA profile** 
    - Yes, there is a note present; not an alert, which is fine. 
### Conditional Fields
- Note: Fix is in for the conditional date, may not happen prior to testing.  Follows the one thing per page better. 
### Custom Work
- Custom - bolded word "**monthly**" all semantic HTML 
- Added custom label and custom hint text legend as a title, and two paragraph tags and use css and classes to style them appropriately, then the actual field for the number. 
    - **Test this with a screen reader.** 
- Experimental design piece on the card for household income.  
### Bugs Identified During Pilot Session
- Bug adding "undefined" in the label in the HTML for some reason.
- Bug with it allowing you to move past required field. 
### Error Issue
- Errors issue ticket for better error messaging, currently inline errors only. 
### Headings
- Heading ordering, avoid skipping levels
- Since using as a legend, instead of H3 above the cards, make the card titles H3 - then you can style as H4, if you want it to appear differently. 
### Note About Fields
- Half use the form system, and other items are custom (summary card page). 
- Most of the fields are standard for system fields.
### Vehicle information section
- Expandable content "Why do I need to provide this information" could be moved up so the "Estimated value" could be closer in proximity to the field for estimated value. 
### Review the ARIA labels
- Relating to the edit and delete buttons (ex. random zero showing up)
### Additional Bugs Encountered
- Tried to add credit card bill (clicked cancel like it wasn't going to be added; and went to a blank screen). 
- Screen is also missing back and continue. 
- It only shows cancel and "add a credit card bill" it should only show on "add additional." 
- This is happening with the installment contracts and other debts, logic is acting funky with the flow. 
### Custom Work Continued
- Debt 1 of 1: Disability compensation and pension debt - these are pretty custom. 
    - Bold line one and second line of text for the description.
    - The front-end folks are working on revamping this completely. 
    - Planning on having the amount of debt in the title, instead of having the text below the title. 
**- Test this with AT once revamped.** 
### Review/Summary Page
_**Review page, has some outstanding tickets they are working to figure that out.**_ 
- (Ex. Employment history isn't showing). 
- The dependent ages fields, FSR acts how many dependents you have and the ages. 
    - Conditional fields that appear based on the number. 
    - **Test this with AT.** 
        - Inquire about how they feel (about the conditional dependent section) since we traditionally avoid that? What do you expect here, if you had one vs. eight dependents?
- Missing "save" for save amount for the trailers and boats. 
- Note: The old system used drop-down fields, and changed this to one-thing per page. 
    - **Review page needs reviewed for functionality.** 
- Some fields are view only on the review page. 
- Submission page worked

### Sharing Screen 
#### Jaws on Windows
- This tends to be a big issue when testing. 
- Move focus to Zoom meeting controls. Hit tab until they hear share screen. Hit space bar and the window should populate. 
#### Share on Mac
- Keyboard shortcut = Command CTRL S 

### View Past Sessions
- View past recordings to see how AT sessions ran. 
- Liz might be a point of contact? SK to reach out.

**Comment by sara-amanda** [comment]
## CAIA Update
### Removing Original Participant 1 from Ticket Body
- **Monday, May 22 11:00am-12:30pm ET** 
        - Assistive Tech: **(Box checked on screener form; no detail provided, despite outreach.)**
        - AT Skill Level: NA
        - Operating System: NA
        - Browser: NA
        - Device Type: P1 registered saying they would use a Smartphone; however, they ended up using a tablet.
       
cc: @SarahKay8 @AngelaFowler82

**Comment by sara-amanda** [comment]
Adding to participant section: 
- [ ] Invite a11y specialist(s) on Zoom invite.

**Comment by sara-amanda** [comment]
## CAIA User Testing Follow-Up
Hi, @charlottecesana @mgleo , I wanted to follow-up to see if you have heard anything from Perigean about future assistive technology users joining your study, since the first participant ended up not using AT? 

**OOO Note:** _I will be out of the office 6/2-6/9, and back on 6/12. If any sessions were to take place during that time, please reach out to @SarahKay8 and @coforma-terry . Terry will be out 6/7-6/9, but @RLHecht of CAIA can be reached during that time._

**Comment by mgleo** [comment]
Hi @sara-amanda, thanks for following up. Unfortunately did not end up speaking with anyone who used AT. The study was closed last week.

**Comment by josephrlee** [comment]
@sara-amanda Also, I realized one of the updates I made, I introduced quite a few nested fieldsets, which [seem to be a no-no](https://adamsilver.io/blog/the-problem-with-nested-fieldsets-and-how-to-avoid-them/). After reading this article, does that sound correct from your perspective? I plan on creating a new PR and reverting the nested fieldsets I introduced.

**Comment by RLHecht** [comment]
Closing this ticket as SK is continuing work in a separate ticket.

**Comment by sara-amanda** [comment]
> Hi @sara-amanda, thanks for following up. Unfortunately did not end up speaking with anyone who used AT. The study was closed last week.

Thanks for the update @mgleo assuming, since this is closed you are not able to extend the study. Will you be looking to do a separate study with AT users in the future?

**Comment by mgleo** [comment]
Hi @sara-amanda, we'd like to speak with several AT users in upcoming studies if we're able to. Unfortunately, I think we might be testing with prototypes in the near future.
cc @charlottecesana
